application/controllers/LoginManager.php

```diff
@@ -18,7 +18,92 @@ function force_ssl() {
 	    // }
 	}
 
+	function handleLocalLoginForVueUI() {
+		$requestMethod = $this->input->server('REQUEST_METHOD');
```

Comment:
If we're breaking this out as a separate method anyways, maybe just move the render back into the localLogin method, then make this method only for the POST result? Then you can skip this request checking.